conductor-client/src/main/kotlin/com/openlattice/directory/Auth0UserDirectoryService.kt

```diff
@@ -42,28 +42,43 @@ class Auth0UserDirectoryService(
     }
 
     //TODO: Switch over to a Hazelcast map to relieve pressure from Auth0
```

Comment:
Just flagging for follow-up, anyone know what the idea here is? 
I'm guessing a mapstore that is backed by querying Auth0?

Reply:
This has been around for over a year, not sure what the intention is, though "a mapstore backed by querying Auth0" makes sense to me.

https://github.com/openlattice/conductor-client/pull/749/files#diff-52f449f6251239dda152342be19580a17f4bee38b4ba62b854b2944eae3f8bf4R35

Reply:
I assume this was from before the users mapstore existed, which now exists, so this can probably be deleted